examples/jetstream/consumers/main.go

```diff
@@ -0,0 +1,181 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/nats-io/nats.go"
+)
+
+/*
+This example shows implementation of consumers management.
+`js-consumer-management` command can be used to create, update or get information about a consumer.
+`-push-target` option can be used to configure a delivery subject for a push-based consumer. If not provided, the consumer will be created in push mode.
+`-filter` option can be used to filter subjects from the stream.
+`-queue` option can be used to create a queue consumer.
+
+Example usage:
+./js-consumer-management -creds '/path/to/nats/credentials' -s 'nats://127.0.0.1:4222' -push-target=ORDERS_RECV add ORDERS ORDERS_CONSUMER
+*/
+
+func usage() {
+	log.Printf("Usage: js-consumer-management [-s server] [-creds file] [-nkey file] [-tlscert file] [-tlskey file] [-tlscacert file] [-push-target subject] [-filter subject] [-queue queue] <operation> [ add | delete | info ] <stream> <name>\n")
+	flag.PrintDefaults()
+}
+
+func showUsageAndExit(exitcode int) {
+	usage()
+	os.Exit(exitcode)
+}
+
+func main() {
+
+	// CLI flags
+	var (
+		urls          = flag.String("s", nats.DefaultURL, "The nats server URLs (separated by comma)")
+		userCreds     = flag.String("creds", "", "User Credentials File")
+		nkeyFile      = flag.String("nkey", "", "NKey Seed File")
+		tlsClientCert = flag.String("tlscert", "", "TLS client certificate file")
+		tlsClientKey  = flag.String("tlskey", "", "Private key file for client certificate")
+		tlsCACert     = flag.String("tlscacert", "", "CA certificate to verify peer against")
+		pushTarget    = flag.String("push-target", "", "Deliverty subject for push-based consumer")
+		filter        = flag.String("filter", "", "FilterSubject for a consumer")
+		queue         = flag.String("queue", "", "Delivery target group for a consumer")
```

Comment:
`queue` only has relevance if `pushTarget` specified and a Push JS Consumer is being created.  Clarify in description.

Reply:
Thank you for pointing that out. I fixed the comment on top of the file as well as the flag description.